packages/cache/src/internal/cacheUtils.ts

```diff
@@ -71,11 +71,13 @@ export async function unlinkFile(filePath: fs.PathLike): Promise<void> {
   return util.promisify(fs.unlink)(filePath)
 }
 
-async function getVersion(app: string): Promise<string> {
+async function getVersion(app: string, args?: string[]): Promise<string> {
```

Comment:
As mentioned in another comment, this need not be an argument. Will simplify the code as well.